src/index.d.ts

```diff
@@ -0,0 +1,63 @@
+import appdmg = require('appdmg');
+
+declare module 'electron-installer-dmg' {
+  namespace createDMG {
+      interface Content {
+          path: string;
+          type: 'link' | 'file';
+          x: number;
+          y: number;
+      }
+
+      interface CreateOptions {
+          /**
+           * Additional options to pass through to [`appdmg`](https://npm.im/appdmg).
+           * You can use this to set additional features like `background-color` and `code-sign`.
+           * See the docs of the `appdmg` module for all possible options.
+           */
+          additionalDMGOptions?: Partial<appdmg.Specification>;
+          /**
+           * The `.app` directory generated by
+           * [electron-packager](https://github.com/electron-userland/electron-packager).
```

Comment:
```suggestion
           * [electron-packager](https://github.com/electron/electron-packager).
```

Reply:
While your suggested change is correct, I'd like to note that this link is outdated in several places in this repository.